.github/workflows/ca.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       matrix:
         php-version:
           - "7.4"
+          - "8.1"
```

Comment:
We would want to switch to PHP 8.1 for static analysis and other common stages, but there are branch protection rules that demand PHP 7.4. So let's add 8.1 first and tweak the protections later.